pymongo/network_layer.py

```diff
@@ -286,6 +287,7 @@ async def _async_socket_receive(
 
 
 _PYPY = "PyPy" in sys.version
+_WINDOWS = os.name == "nt"
```

Comment:
Our usual windows check is `sys.platform == "win32"`

Reply:
Updated